src/modules/dashboard/components/ManageFundsDialog/ManageFundsDialog.tsx

```diff
@@ -106,14 +107,17 @@ const ManageFundsDialog = ({
   nextStepMintTokens,
   nextStepManageTokens,
   nextStepUnlockToken,
+  isVotingExtensionEnabled,
 }: Props) => {
   const { walletAddress, username, ethereal } = useLoggedInUser();
 
   const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
 
   const hasRegisteredProfile = !!username && !ethereal;
   const canMoveFunds = hasRegisteredProfile && canFund(allUserRoles);
-  const canMintNativeToken = colony.canMintNativeToken && hasRoot(allUserRoles);
+  const canMintNativeToken =
+    (colony.canMintNativeToken && hasRoot(allUserRoles)) ||
+    isVotingExtensionEnabled;
```

Comment:
While the logic is good, I think this is hard to read.

I would have left the `canMintNativeToken` logic as it is, and just change the check in the object:
```
    {
      title: MSG.mintTokensTitle,
      description: MSG.mintTokensDescription,
      icon: 'emoji-seed-sprout',
      permissionRequired: !isVotingExtensionEnabled || !canMintNativeToken,
      permissionInfoText: MSG.permissionsListText,
      permissionInfoTextValues: {
        permissionsList: (
          <FormattedMessage {...MSG.mintTokensPermissionsList} />
        ),
      },
      onClick: () => callStep(nextStepMintTokens),
    },
```